spaces/S000088/properties/P000174.md

```diff
@@ -0,0 +1,11 @@
+---
+space: S000088
+property: P000174
+value: true
+---
+
+Clearly the points $a^\pm$ admit ordered bases.
```

Comment:
```suggestion
Clearly the points $a^\pm$ admit totally ordered local bases.
```
Also, I know we use `a^\pm` all over the place in this space.  But is that really clearer than just saying "the points $a^+$ and $a^-$ ..." for example?